boards/pba-d-01-kw2x/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
sorry for the confusion!
<pre>
ifneq (,$(filter kw2xrf,$(USEMODULE)))
USEMODULE+=ng_ieee802154
</pre>
should go in mkefile.ep, but we can fix that once #2947 is merged. the concept is not fixed, yet.